_posts/2019-01-23-lecture-03.md

```diff
@@ -0,0 +1,470 @@
+---
+layout: distill
+title: "Lecture 3: Undirected Graphical Models"
+description: An introduction to undirected graphical models
+date: 2019-01-23
+
+lecturers:
+  - name: Eric Xing
+    url: "https://www.cs.cmu.edu/~epxing/"
+
+authors:
+  - name: Allan Wang
+    url: "#"  # optional URL to the author's homepage
+  - name: Binghao Deng
+    url: "#"
+  - name: Weizhao Shao
+    url: "#"
+  - name: Zhuoran Liu
+
+editors:
+  - name: Lisa Lee
+
+abstract: >
+---
+
+## Review
+
+In addition to the **I-map** concept that is introduced in the last lecture, today's lecture also includes **minimal I-map**.
+
+### Definition: Minimal I-maps
+A DAG is a **minimal I-map** if it is an I-map for $$P$$, and if the removal of even a single edge from $$\mathcal{G}$$ renders it not an I-map.
+
+* A distribution may have several minimal I-maps. Each corresponding to a specific node-ordering.
+
+* The fact that $$\mathcal{G}$$ is a minimal I-map for $$P$$ is **far from** a guarantee that $$\mathcal{G}$$ captures the independence structure in $$P$$.
+
+## "Bayes-ball" Algorithm
+
+"Bayes-ball" algorithm is an algorithm that we can apply to retrieve independences directly from the graphical models. It states that $$X$$ is **d-separated** from $$Z$$ given $$Y$$ if we cannot send a ball from any node in $$X$$ to any node in $$Z$$. "Given" is represented by shading the node in the actual graph. Examples of three basic directed graphical structures are shown below.
+
+<figure id="basic-structure" class="l-body">
+    <div class="row">
+        <div class="col two">
+            <img src="{{ '/assets/img/notes/lecture-03/basic.png' | relative_url }}" />
+        </div>
+    </div>
+</figure>
+
+* From the first and the second structure, we can see that the shaded $$Y$$ node blocks the ball from going between node $$X$$ and $$Z$$, which leads to the independence relations that have been introduced in the last lecture: $$X \perp Z \mid Y$$.
+
+* The third one, the "V-structure", is a special case. Opposite from the first two exmaples, the ball can go between $$X$$ and $$Z$$ if the node $$Y$$ is shaded and is blocked otherwise. Therefore, the graph on the right will yield $$X \perp Z \mid Y$$.
+
+With the basic structures, we can try to apply the rules on a DAG. For example, let us try to find whether $$X_2$$ and $$X_3$$ are independent given $$X_1$$ and $$X_6$$.
+
+<figure id="bayes-ball-example" class="l-body">
+    <div class="row">
+        <div class="col two">
+            <img src="{{ 'assets/img/notes/lecture-03/dag.png' | relative_url }}" />
+        </div>
+    </div>
+</figure>
+
+After shaded $$X_1$$ and $$X_6$$, we can see that the ball cannot go from $$X_2$$ to $$X_3$$ through $$X_1$$ because it is blocked; however, $$X_2$$, $$X_5$$ and $$X_6$$ forms a "V-structure", so the ball can go between $$X_2$$ and $$X_3$$ through the shaded $$X_6$$. Therefore, the independence statement is invalid.
```

Comment:
```suggestion
After shading $$X_1$$ and $$X_6$$, the ball cannot go from $$X_2$$ to $$X_3$$ through $$X_1$$ because it is blocked; however, $$X_2$$, $$X_6$$, and $$X_5$$ forms a "V-structure", so the ball can go along the path $$X_2$$, $$X_6$$, $$X_5$$, $$X_3$$. Therefore, the independence statement is invalid.
```